addons/website/views/website_templates.xml

```diff
@@ -91,7 +91,7 @@
         <t t-set="website_meta" t-value="seo_object and seo_object.get_website_meta() or {}"/>
         <meta name="default_title" t-att-content="default_title" groups="website.group_website_designer"/>
         <meta t-if="(main_object and 'website_indexed' in main_object
-        and not main_object.website_indexed) or (website.domain and website.domain not in request.httprequest.url_root)" name="robots" content="noindex"/>
+        and not main_object.website_indexed) or (website.domain and not website._is_indexable_url(request.httprequest.url_root))" name="robots" content="noindex"/>
```

Comment:
>I wanted to merge but then saw it was targeting master, why? Did we discuss something I forgot?
This is quite a critical fix (it prevents wiping completely your website from crawlers/indexation), please fix it in the earliest supported version possible.
Approving this, just let me know when target is changed

@rdeodoo Isn't it a problem to do a change in this file in stable ? :thinking:

Reply:
No, worst case some people won't have the fix without updating website app.
But it won't break anything else, what we can't do is remove attribute or class as it will break xpath.
Here, it's just about modifying business code logic inside a `t-if` condition, no one will have written a xpath being `[t-if:contains("website.domain not in request.httprequest.url_root")]`. And even if they did, it's out of what we support, they should use class (contain and not equal) or attribute.